plugins/woo-ai/includes/class-woo-ai-settings.php

```diff
@@ -168,74 +96,58 @@ public function add_woo_ai_settings( $settings = array(), $current_section = nul
-			),
-
-			array(
-				'title'   => __( 'Enable AI', 'woocommerce' ),
-				'desc'    => __( 'Enable AI features in your store', 'woocommerce' ),
-				'id'      => 'woo_ai_enable_checkbox',
-				'default' => 'yes',
-				'type'    => 'checkbox',
+		$settings_ai[] = array(
+			'id'      => self::TONE_OF_VOICE_OPTION_KEY,
+			'name'    => __( 'Tone of voice', 'woocommerce' ),
+			'desc'    => __( 'Select the tone of voice for the AI', 'woocommerce' ),
+			'type'    => 'select',
+			'options' => array(
+				'informal'     => __( 'Relaxed and friendly.', 'woocommerce' ),
+				'humorous'     => __( 'Light-hearted and fun.', 'woocommerce' ),
+				'neutral'      => __( 'A balanced tone that uses casual expressions.', 'woocommerce' ),
+				'youthful'     => __( 'Friendly and cheeky tone.', 'woocommerce' ),
+				'formal'       => __( 'Direct yet respectful formal tone.', 'woocommerce' ),
+				'motivational' => __( 'Passionate and inspiring.', 'woocommerce' ),
 			),
+			'css'     => 'min-width:300px;',
 
-			array(
-				'title'             => __( 'Tone of voice', 'woocommerce' ),
-				'id'                => self::TONE_OF_VOICE_OPTION_KEY,
-				'css'               => 'min-width:300px;',
-				'default'           => 'neutral',
-				'type'              => 'select',
-				'desc'              => $markup_str,
-				'desc_tip'          => __( 'Choose the language style that best resonates with your customers. It\'ll be used in text-based content, like product descriptions.', 'woocommerce' ),
-				'custom_attributes' => array( 'disabled' => 'true' ),
-				'options'           => $this->tone_of_voice_select_options,
-			),
+		);
 
-			array(
-				'id'                => self::STORE_DESCRIPTION_OPTION_KEY,
-				'type'              => 'textarea',
-				'custom_attributes' => array( 'disabled' => 'true' ),
-				'title'             => __( 'Describe your business', 'woocommerce' ),
-				'desc_tip'          => __( 'Tell us what makes your business unique to further improve accuracy of the AI-generated content. This will not be shown to customers.', 'woocommerce' ),
-				'placeholder'       => __( 'e.g. Marianne Renoir is a greengrocery taken over by a ten generations Parisian family who wants to keep quality and tradition in the quarter of Montmartre', 'woocommerce' ),
-				'css'               => 'min-width:300px;min-height: 130px;',
-			),
+		$settings_ai[] = array(
+			'id'          => self::STORE_DESCRIPTION_OPTION_KEY,
+			'title'       => __( 'Describe your business', 'woocommerce' ),
+			'type'        => 'textarea',
+			'desc_tip'    => __( 'Tell us what makes your business unique to further improve accuracy of the AI-generated content. This will not be shown to customers.', 'woocommerce' ),
+			'placeholder' => __( 'e.g. Marianne Renoir is a greengrocery taken over by a ten generations Parisian family who wants to keep quality and tradition in the quarter of Montmartre', 'woocommerce' ),
```

Comment:
I think a short doc block for the translators (`/* translators: ... */`) would be good here. Just to indicate what this sentence is and how it will be used.